Framework/Kernel/src/Unit.cpp

```diff
@@ -583,6 +583,54 @@ double dSpacing::conversionTOFMax() const { return DBL_MAX / factorTo; }
 
 Unit *dSpacing::clone() const { return new dSpacing(*this); }
 
+// ==================================================================================================
+/* D-SPACING Perpendicular
+ * ==================================================================================================
+ *
+ * Conversion uses equation: d^2 = lambda^2 - 2[Angstrom^2]*ln(sin(theta))
```

Comment:
Just add a detailed description (with formula) or a paper reference, to show what this unit is all about. What I mean is that I want to be able to check that the formula that you implement makes sense or not